### PR TITLE
implementing files facet

### PR DESCRIPTION
**Pull request by truhacevkir**
Making this much easier by populating files enum.

### CODE REVIEW

**Review by StigNorland — DISMISSED** [review]

**Review by StigNorland — DISMISSED** [review]

File: search-commons/src/test/resources/resource_urls.json [hunk 1]
```diff
@@ -53,16 +53,8 @@
...
-  "HAS_PUBLIC_FILE": 20,
-  "HAS_FILE=false": 1,
-  "HAS_FILE=PublishedFile": 20,
-  "HAS_PUBLIC_FILE=0": 1,
-  "HAS_PUBLIC_FILE=false": 1,
-  "NO_PUBLIC_FILE": 1,
-  "NO_PUBLIC_FILE=noPublicFile": 1,
-  "NO_PUBLIC_FILE=true": 1,
+  "files=hasPublicFiles": 19,
+  "files=noFiles": 2,
```
Comment (StigNorland) [inline search-commons/src/test/resources/resource_urls.json:57]: nice!

File: search-commons/src/test/java/no/unit/nva/search2/ResourceClientTest.java [hunk 2]
```diff
@@ -328,6 +328,32 @@ void searchWithUriReturnsOpenSearchAwsResponse(URI uri, int expectedCount) throw
...
+            if (expectedCount == 0) {
+                logger.debug(pagedSearchResourceDto.toJsonString());
+            } else {
+                logger.debug(pagedSearchResourceDto.toString());
+            }
+
+            assertThat(pagedSearchResourceDto.hits().size(), is(equalTo(expectedCount)));
+            assertThat(pagedSearchResourceDto.totalHits(), is(equalTo(expectedCount)));
+        }
+
```
Comment (StigNorland) [inline search-commons/src/test/java/no/unit/nva/search2/ResourceClientTest.java]: isn't this just a copy of another test with the same input?

**Review by StigNorland — CHANGES_REQUESTED** [review]
I really liked what you did here. much simpler!
just explain for me what the test was all about...

File: search-commons/src/test/java/no/unit/nva/search2/ResourceClientTest.java (hunk 2 above)
Comment (truhacevkir) [reply search-commons/src/test/java/no/unit/nva/search2/ResourceClientTest.java]: Midlertidig test :), fjerner denne.

**Review by StigNorland — APPROVED** [review]

**Pull request merged**